### PR TITLE
Add before picture for comment box

### PR DESCRIPTION
**Pull request by raghavrv**
Apparently, I've been using this for quite a while and I had completely forgotten I installed it in the first place!

It would be nice to have a "how it was" view ;)

### CODE REVIEW

**Comment by hkdobrev** [comment]
👍

Why do you need the arrow image?

**Comment by raghavrv** [comment]
Maybe not... I'll remove... Thx

File: readme.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ And [lots](extension/content.css) [more...](extension/content.js)
 
 ### Comment box
 
-<img src="screenshot-comment-box.png" width="795">
+![](screenshot-comment-box-before.png) ![](screenshot-comment-box-after.png)
```
Comment (hkdobrev) [inline readme.md]: Could you also add before/after labels above the images? Thank you!

**Comment by sindresorhus** [comment]
The images needs to be retina though.

**Comment by paulmolluzzo** [comment]
@raghavrv 👋  friendly ping.

**Pull request closed without merging**